app/controllers/course/stage/instructions.ts

```diff
@@ -82,7 +82,7 @@ export default class CourseStageInstructionsController extends Controller {
 
     // If the "active step" is after a "BaseStagesCompletedStep", navigate to "BaseStagesCompletedStep" instead.
     if (nextStep?.type === 'BaseStagesCompletedStep' && stepAfterNextStep === activeStep) {
-      this.router.transitionTo(nextStep.routeParams.route, nextStep.routeParams.models);
+      this.router.transitionTo(nextStep.routeParams.route, nextStep.routeParams.models[0]!);
```

Comment:
```suggestion
      // @ts-expect-error
      this.router.transitionTo(nextStep.routeParams.route, ...nextStep.routeParams.models);
```

`@ts-expect-error` since it looks like Ember's types don't allow variadic arguments yet. `...models` since that's doesn't include an assumption around how many arguments the route has

Reply:
Hmm, actually I'm confused now - how does the invocation above on line 78 work? 

```
this.router.transitionTo(activeStep.routeParams.route, activeStep.routeParams.models);
```

Shouldn't that have this same problem too? 🤔

Reply:
in the previous line, the route needs two models, course and course stage. tried digging in issues on the ember repo but i couldn't find out why when a route expects a single model that an array of a single identifier doesn't work